wa/framework/target/runtime_config.py

```diff
@@ -930,6 +935,17 @@ def initialize(self):
                     Specify whether the device screen should be on
                     """)
 
+            param_name = 'unlock_screen'
+            self._runtime_params[param_name] = \
+                RuntimeParameter(
+                    param_name, kind=str,
+                    default='vertical',
```

Comment:
I'm open to feedback about the default value/behavior.
Setting default value to empty string to disable unlocking screen feature also makes sense in order to keep current behavior of workload-automation.

Reply:
> Setting default value to empty string

Isn't there a way to have `None` by default (`null` in YAML) ? I'm not familiar with that parameter system.

Reply:
`None` is kind of equal to empty string.
I wonder if we should disable unlocking screen feature to preserve current behavior of WA, or enable it by default.

Reply:
Agreed, to be consistent with the other parameters we should use `None` here rather than an empty string.
Also I don't think we should change the existing default behaviour, WA recommends that the screen lock is disabled by default so in an ideal case we should not have to try unlocking the device and therefore have less setup steps.